client/src/pages/SubmitPage/Submit.tsx

```diff
@@ -80,6 +83,13 @@ function FileUploader(): JSX.Element {
                 updateSnackBarMessage(
                   `Submitted ${imageList.length} images to ${contest == undefined ? 'Contest' : contest.title}`,
                 );
+                {
+                  contest.userID == undefined
```

Comment:
What is the reason for the ternary operator here? If contest.userID is undefined won't we always be return a loading spinner?

Reply:
Yes I defined contest.userID == undefined at the top to return a loading spinner but for some odd reasons typescript still complains contest.userID is undefined

Reply:
Maybe check your interfaces as well as how your passing a contest into this component?